jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessor.java

```diff
@@ -183,7 +183,8 @@ static PluginConfigurationProcessor processCommonConfiguration(
 
     JibContainerBuilder jibContainerBuilder =
         projectProperties
-            .createContainerBuilder(baseImage)
+            .createContainerBuilder(
+                baseImage, getAppRootChecked(rawConfiguration, projectProperties))
```

Comment:
Maybe it doesn't matter much, but we're checking the approot twice now (once here and once in `computeEntrypoint`. I wonder if we should just call `getAppRootChecked()` once and pass the result into both `createContainerBuilder` and `computeEntrypoint`.

Reply:
Valid point, but `computeEntrypoint()` should still accept both `rawConfiguration` and `projectProperties`, and it's cheap to call `getAppRootChecked()`. I think it's better than adding another argument to pass to `computeEntrypoint()` from the API perspective. No need to make caller to have `appRoot` computed when the method can get it easily. Also prevents passing an inconsistent `appRoot` value by accident.